packages/p2p/src/components/p2p-empty/p2p-empty.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Icon, Text } from '@deriv/components';
+import { Icon } from '@deriv/components';
 
 type TP2pEmptyProps = {
     className?: string;
     children: React.ReactNode;
     has_tabs?: boolean;
     icon: string;
-    title: string;
+    is_disabled?: boolean;
+    title: React.ReactNode;
 };
 
-const P2pEmpty = ({ className, children, has_tabs = false, icon, title }: TP2pEmptyProps) => {
+const P2pEmpty = ({ className, children, has_tabs = false, is_disabled = false, icon, title }: TP2pEmptyProps) => {
+    const is_disabled_color = is_disabled ? 'disabled' : '';
+
     return (
         <div className={classNames(className, 'p2p-empty', { 'p2p-empty--no-tabs': !has_tabs })}>
-            <Icon icon={icon} className='p2p-empty-icon' size={128} />
-            <div className='p2p-empty-title'>
-                <Text weight='bold'>{title}</Text>
-            </div>
```

Comment:
should we remove this? cause this can affect other pages where the text is bold like in `advertiser-page-adverts.jsx` when the advertiser has no adverts, or in `order-table-content.jsx`

<img width="818" alt="Screenshot 2023-12-21 at 9 52 13 AM" src="https://github.com/nada-deriv/deriv-app/assets/103412909/d0e8b50c-a81a-4c9a-8c44-9ee480865c2b">

<img width="616" alt="Screenshot 2023-12-21 at 9 53 42 AM" src="https://github.com/nada-deriv/deriv-app/assets/103412909/c764b6c8-eed8-40b9-afe7-f80a9d57e573">

Reply:
We are already passing bold from order-table-content but not in advertiser-page-adverts. Nice catch!